iceoryx_posh/include/iceoryx_posh/internal/roudi/process.hpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
This is not a new file. Finally managed to nicely split a file with ful history :sunglasses: Try `git blame` locally.